### PR TITLE
Exams by Evgheniy Kovaliov IM-21

### PR DESCRIPTION
**Pull request by kovaliovev**
Refactored except.js and password.js

### CODE REVIEW

**Review by Vladyulik — COMMENTED** [review]
Excellent code! Great work

**Review by sonakrrr — COMMENTED** [review]
everything is great!

File: Tasks/password.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-let GeneratePassword = (alphabet, length) => {
-  const MAX = alphabet.length;
-  key = '';
+const generatePassword = (alphabet, length) => {
+  const max = alphabet.length;
+  let password = '';
   for (let i = 0; i < length; i++) {
-    Index = Math.floor(Math.random() * MAX);
-    key = key + alphabet[Index];
+    const index = Math.floor(Math.random() * max);
```
Comment (tshemsedinov) [inline Tasks/password.js:8]: It's better to extract `random` as a separate function

Comment (kovaliovev) [reply Tasks/password.js:8]: was fixed here https://github.com/kovaliovev/Exams/blob/exams/Tasks/password.js

**Pull request closed without merging**